net/suricata/Config.in

```diff
@@ -0,0 +1,36 @@
+menu "Configuration"
+	depends on PACKAGE_suricata
+
+config SURICATA_LOGROTATE
+	bool "Copy Suricata logrotate file"
+	default n
+
+config SURICATA_DEBUG
+	bool "Enable debug output"
+	default n
+
+config SURICATA_LUA
+	bool "Enable Lua support"
+	default n
+
+config SURICATA_MAGIC
+	bool "Enable Libmagic support"
+	default y
+
+config SURICATA_NFLOG
+	bool "Enable libnetfilter_log support"
+	default n
+
+config SURICATA_NFQ
+	bool "Enable NFQUEUE support for inline IDP"
+	default y
+
+config SURICATA_PIE
+	bool "Enable compiling as a position independent executable"
+	default n
```

Comment:
this shouldn't be an enduser option, surely?

Reply:
I've tried to expose most of the config args without considering my personal interest. :)
I'll remove/hardcode/change any of them if there is a solid decision on what their values must be.